rust/bridge/shared/src/lib.rs

```diff
@@ -744,6 +745,13 @@ bridge_get!(SessionRecord::local_registration_id -> u32);
 bridge_get!(SessionRecord::remote_registration_id -> u32);
 bridge_get!(SessionRecord::has_sender_chain as HasSenderChain -> bool, ffi = false, node = false);
 
+bridge_get!(SealedSenderDecryptionResult::sender_uuid -> String, ffi = false, jni = false);
+bridge_get!(SealedSenderDecryptionResult::sender_e164 -> Option<String>, ffi = false, jni = false);
```

Comment:
You should be able to use `&str` now (to avoid a copy)…

Reply:
A little WTF here, I changed to &str, works great locally, but CI fails

```
error[E0277]: the trait bound `Option<&str>: ffi::convert::ResultTypeInfo` is not satisfied
   --> rust/bridge/shared/src/support/mod.rs:94:13
    |
94  |             #[bridge_fn($($param = $val),*)]
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `ffi::convert::ResultTypeInfo` is not implemented for `Option<&str>`
    | 
   ::: rust/bridge/shared/src/lib.rs:549:1
    |
549 | bridge_get!(SenderCertificate::sender_e164 -> Option<&str>);
    | ------------------------------------------------------------ in this macro invocation
    |
```

Reply:
Oops, guess I never added that for result types in FFI. It shouldn't be too hard to add, but you don't have to do that now.